package.json

```diff
@@ -218,6 +218,7 @@
     "@kbn/es-errors": "link:bazel-bin/packages/kbn-es-errors",
     "@kbn/es-query": "link:bazel-bin/packages/kbn-es-query",
     "@kbn/eslint-plugin-imports": "link:bazel-bin/packages/kbn-eslint-plugin-imports",
+    "@kbn/eslint-plugin-packages": "link:bazel-bin/packages/kbn-eslint-plugin-packages",
```

Comment:
I think this responsibility of this package it to house the eslint rules focused on eslint rules, so I think we should name it `@kbn/eslint-plugin-rules`, maybe `@kbn/eslint-plugin-disable`